cLIMS/organization/importSeqFiles.py

```diff
@@ -49,7 +53,7 @@ def importSeqFiles(request,pk):
             if(filePath==True and v[filePathIndx]!=""):
                 path=v[filePathIndx]
             
-            if(expName!= "" and path!= ""):
+            if(expName!= "" and path!= ""):   
```

Comment:
couple of spaces at the end of the line - again , see 1st comment
we need to cleanup to simplify the code review